### PR TITLE
fix(hybrid): Make SentrySwizzle available for HybridSDKs

### PR DESCRIPTION
**Pull request by krystofwoldrich**
## :scroll: Description

The `SentrySwizzle` is needed for frames tracking on Hybrid SDKs.

- https://github.com/getsentry/sentry-react-native/pull/3588

## :green_heart: How did you test it?
local build, ci

## :pencil: Checklist

You have to check all boxes before merging:

- [x] I reviewed the submitted code.
- [ ] I added tests to verify the changes.
- [x] No new PII added or SDK only sends newly added PII if `sendDefaultPII` is enabled.
- [ ] I updated the docs if needed.
- [ ] Review from the native team if needed.
- [x] No breaking change or entry added to the changelog.
- [x] No breaking change for hybrid SDKs or communicated to hybrid SDKs.

### CODE REVIEW

**Comment by philipphofmann** [comment]
@krystofwoldrich, your approach of using swizzling for the starting the frames tracker looks a bit weird to me here https://github.com/getsentry/sentry-react-native/pull/3588. I don't think you need that to start the tracker. I added a comment to the PR https://github.com/getsentry/sentry-react-native/pull/3588#discussion_r1505744625.

**Review by brustolin — APPROVED** [review]
Before merging this, take a look at Philipps comment, maybe there is another solution for what you need, otherwise LGTM.

**Comment by krystofwoldrich** [comment]
Thanks, I haven't found a different solution. I'm going to merge this.

**Pull request merged**
